pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -827,5 +827,20 @@ public static void verifyHybridTableConfigs(String rawTableName, TableConfig off
           "Time column names are different for table: %s! Offline time column name: %s. Realtime time column name: %s",
           rawTableName, offlineTimeColumnName, realtimeTimeColumnName));
     }
+    TenantConfig offlineTenantConfig = offlineTableConfig.getTenantConfig();
+    TenantConfig realtimeTenantConfig = realtimeTableConfig.getTenantConfig();
+    String offlineBroker = offlineTenantConfig.getBroker();
+    String realtimeBroker = realtimeTenantConfig.getBroker();
+    if (offlineBroker == null || realtimeBroker == null) {
+      throw new IllegalStateException(String.format(
```

Comment:
I hope these get translated to 4xx error and not 5xx. I would think of IllegalStateException as something that is wrong on the server end. These are clearly bad user inputs causing this exception, so it should be IllegalArgumentException ? I understand you are following the pattern from before, but if you could dig on this a bit and fix it that will be awesome.

Thanks.

Reply:
Thanks @mcvsubbu got you points. I will fix it.